src/components/NavPanel.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react';
+import { Nav, NavItem } from 'react-bootstrap';
+import styles from './../../public/stylesheets/nav.css';
+
+
+const NavPanel = ({ active }) => (
+  <div className={styles.navpanel}>
+    <Nav bsStyle="pills" stacked activeKey={active} >
+      <NavItem eventKey={1} className={styles.item} href="/">Home</NavItem>
+      <NavItem eventKey={2} className={styles.item} href="#">Create/Edit Plan of Study</NavItem>
```

Comment:
It's only there to be an example; links that don't redirect you will look like the one below